slides/07_Data_Fetching.html

```diff
@@ -438,15 +362,41 @@ <h2>Exercise</h2>
                 >.
               </li>
               <li>
-                API call:
+                Install
+                <a
+                  href="https://tanstack.com/query"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  >TanStack Query</a
+                >: <span class="code">npm i @tanstack/react-query</span>
+              </li>
+              <li>
+                API call with SWR:
                 <span class="code"
                   >GET https://pokeapi.co/api/v2/pokemon?limit=50</span
                 >
               </li>
+              <li>Stretch goal: Display loading & error messages
```

Comment:
The stretch goal is cut off on my screen. Unfortunately reveal.js is not very flexible there. I would recommend to add another slide for the stretch goal or split the exercise slides somehow differently